DeAceroDLabs/marketplace-frontend
# [DMP-718] 404 error on productdetail and search

## What is done

[[DMP-718] 404 error on productdetail and search](https://deacero.atlassian.net/browse/DMP-718)

## How is done

- Change search route to /buscar .
- The problem was that proxy was doing a get on the url, now it is done inside the views.
- Added a quick fix to the page title (Maria approved it):

![image](https://user-images.githubusercontent.com/81311646/205140854-c863a78b-904f-4a45-883b-a9eaaddda172.png)


## How to test

1. Run the project via `npm start`.
2. Click on any product.
3. Refresh the page. The page should work as expected.
4. Search some product, and refresh.
5. Right now the search service is not working but as soon as it works this should work as expected, before we found a 404 error and now it stays loading.

.
